src/datastore/postgres-store.ts

```diff
@@ -3762,6 +3767,70 @@ export class PgDataStore extends (EventEmitter as { new (): DataStoreEventEmitte
     });
   }
 
+  async updateBatchTokenOfferingLocked(client: ClientBase, lockedInfos: DbTokenOfferingLocked[]) {
+    const columnCount = 3;
+    const insertParams = this.generateParameterizedInsertString({
+      rowCount: lockedInfos.length,
+      columnCount,
+    });
+    const values: any[] = [];
+    for (const lockedInfo of lockedInfos) {
+      values.push(lockedInfo.address, lockedInfo.value, lockedInfo.block);
+    }
+    const insertQuery = `INSERT INTO token_offering_locked (
+      address, value, block
+      ) VALUES ${insertParams}`;
+    const insertQueryName = `insert-batch-subdomains_${columnCount}x${lockedInfos.length}`;
```

Comment:
Just saw this, the query name can't be a duplicated (`insert-batch-subdomains`)

Reply:
fixed this one.